<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>393</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>559</width>
    <height>393</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CalibMonitor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/burn.png</normaloff>images/burn.png</iconset>
  </property>
  <widget class="QLabel" name="label_volume_add">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume_sub">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_pause_play">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProductName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Nameï¼š</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_ProductName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Label_VendorId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vendor ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProductID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_VendorId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>301</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="text_ProductID">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_connect_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/disconnected.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_SwVer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SW Ver:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_SwVer">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_SN">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SN:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_SN">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="key_add_vol">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>41</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mv</string>
   </property>
  </widget>
  <widget class="QLabel" name="key_pause_play_vol">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>118</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>119</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mv</string>
   </property>
  </widget>
  <widget class="QLabel" name="key_sub_vol">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>198</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>199</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mv</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
